rts/at/bridge/src/main/java/org/jboss/narayana/rest/bridge/inbound/InboundBridge.java

```diff
@@ -212,11 +222,15 @@ private void enlist(final InboundBridge inboundBridge) {
      *
      * @return
      */
-    private Transaction getTransaction() {
+    private Transaction getTransaction(boolean useContainerToLookup) {
```

Comment:
@mmusgrov : my understanding is that there is no scenario where txbridge is not running in WFLY

@tomjenkinson : from my testing yes, there is need your handling. There is some trouble with synchronization as you talked with David before. But maybe David's changes in wfly txn client could fix that ...?